### DISCUSSION TITLE
Need clarification on 1.4.10: Reflow

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#2867] [Q&A] [answered]
Hi Team,

I need clarification on  1.4.10: Reflow checkpoint. 

When I check on laptop/desktop after applying the reflow settings, horizontal scrolling appears for the web page.
When I checked the same web page in the actual mobile device browser, I can view all the content without scrolling horizontally.
Can we consider it as a pass here?

**Comment by patrickhlauke** [comment, marked as answer]
Reflow isn't about mobile. It's aimed at desktop users that need increased magnification.

Also, *if* the layout is different between desktop at high magnification, and on an actual mobile device, it sounds like there's some further change happening - browser sniffing or similar? In which case you'll need to evaluate both variants (for all other SCs)
